spec/fx/definition_spec.rb

```diff
@@ -28,6 +28,31 @@
           %r(Define function in db/functions/test_v01.sql before migrating),
         )
       end
+
+      context "when definition is at Rails engine" do
+        let(:engine_path) { Rails.root.join("tmp", "engine") }
```

Comment:
These were forgotten to be moved too.

Reply:
Damn, didn't notice :(
I'm not used to this approach.
It's fixed now! You might take a look again.

Reply:
No worries!